Formula/sqlcipher.rb

```diff
@@ -34,13 +34,27 @@ def install
     ]
 
     if build.with?("fts")
-      args << "CFLAGS=-DSQLITE_HAS_CODEC -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS -DSQLITE_ENABLE_FTS5"
+      args << "CFLAGS=-DSQLITE_HAS_CODEC -DSQLITE_ENABLE_JSON1 -DSQLITE_ENABLE_FTS3 -DSQLITE_ENABLE_FTS3_PARENTHESIS -DSQLITE_ENABLE_FTS5"
```

Comment:
Hmmm, so is the existing `-DSQLITE_HAS_CODEC`.  Might as well put that in the array above too...

Reply:
Hmm I guess we can't since this is actually a single argument. :see_no_evil:

Reply:
Heh Heh Heh, just switched to my OSX desktop to get this done... and yeah.  Oops. :smile: